src/api/libcellml/units.h

```diff
@@ -564,7 +564,7 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
      */
     std::string unitId(size_t index) const;
 
-#ifdef DAMN_YOU_EMSCRIPTEN
+#ifdef EMSCRIPTEN_JAVASCRIPT_BINDINGS
```

Comment:
Why not just `JAVASCRIPT_BINDINGS`? Whether we use Empscripten should be irrelevant here?